### ISSUE TITLE
Can't locate Text/ANSI/Util.pm in @INC

### ISSUE THREAD

**Issue by eserte** [perlancar/perl-Text-Table-More#2]
Another module seems to be undeclared with 0.11:
```
#   Failed test 'No tests run for subtest "wide_char & color"'
#   at t/01-basic.t line 367.
Can't locate Text/ANSI/Util.pm in @INC (you may need to install the Text::ANSI::Util module) (@INC contains: ...  .) at /home/cpansand/.cpan/build/2021022105/Text-Table-More-0.011-vVErKc/blib/lib/Text/Table/More.pm line 175.
# Tests were run but no plan was declared and done_testing() was not seen.
# Looks like your test exited with 255 just after 1.
t/01-basic.t ............. 
Dubious, test returned 255 (wstat 65280, 0xff00)
Failed 1/1 subtests 
```

**Comment by eserte** [comment]
Another one is undeclared:
```
Can't locate Text/ANSI/WideUtil.pm
```

**Comment by perlancar** [comment]
Fixed in 0.013, thanks.
